app/experimenter/nimbus-ui/src/components/HeaderExperiment/index.tsx

```diff
@@ -74,17 +91,25 @@ const StatusPill = ({
   label,
   active,
   padded = true,
+  weight = "primary",
+  className = "",
 }: {
   label: string;
   active: boolean;
   padded?: boolean;
+  weight?: string;
```

Comment:
Nit: `weight` could be a little unclear; since it's only applicable to the active class maybe `activeVariant` could be clearer (`variant` is an established prop in React Bootstrap). What do you think?

Reply:
Yeah I wasn't sure what to call it

Reply:
Actually looking at the bootstrap docs probably `color` is right